man/man1/zhack.1

```diff
@@ -71,6 +71,44 @@ feature in \fIpool\fR.
 .IP
 The \fB\-m\fR switch indicates that the \fIguid\fR feature is now
 required to read the pool MOS.
+.LP
+.BI "scrub " "pool"
```

Comment:
One thing which isn't clear to me after reading this is what the expected default behavior is without any flags.  It would be nice to align this with the `zpool scrub` command as much as possible.  So I would expect `zhack scrub` to either start a new scrub, or resume any running scrub.  We've also had times in the past where it would have been nice to have these very basic options, what do you think about adding them.

**-s** - Stop the running scrub and cleanly export the pool. (new)
**-p** - Pause the running scrub and cleanly export the pool. (new)

Reply:
I'm not opposed, but that sounds like learning more of the ZFS API.  I won't try to stop someone else from adding them later, tho'.

Reply:
I agree that the syntax should be as close to the zpool scrub syntax as possible, for maximum usability. Pause is very simple to implement, as is stop; see `zfs_ioc_pool_scan`.